app/src/main/java/com/framgia/soundclound/service/MusicService.java

```diff
@@ -0,0 +1,290 @@
+
+        Intent nextIntent = new Intent(getApplicationContext(), PlayTrackActivity.class);
+        nextIntent.setAction(Constant.ACTION_NEXT);
+        PendingIntent pnextIntent = PendingIntent.getService(getApplicationContext(),
+                0,
+                nextIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        Notification.Builder builder = new Notification.Builder(getApplicationContext());
+        builder.setContentIntent(pendInt)
+                .setSmallIcon(android.R.drawable.ic_media_play)
+                .setTicker(songTitle)
+                .setOngoing(true)
+                .setContentTitle("hihi")
+                .setContentText("title")
+                .addAction(android.R.drawable.ic_media_previous, "Previous",
+                        ppreviousIntent)
+                .addAction(android.R.drawable.ic_media_play, "Play",
+                        pplayIntent)
+                .addAction(android.R.drawable.ic_media_next, "Next",
+                        pnextIntent);
+        Notification not = builder.build();
+        startForeground(Constant.FOREGROUND_SERVICE, not);
+    }
+
+    public int getPosition() {
+        return player.getCurrentPosition();
+    }
+
+    public boolean isShuffle() {
+        return mShuffle;
+    }
+
+    public boolean isRepeat() {
+        return mRepeat;
+    }
+
+    @Override
+    public boolean isPlay() {
+        return player.isPlaying();
+    }
+
+    @Override
+    public void play() {
+        player.reset();
+        Track track = mTracks.get(curentTrack);
+        songTitle = track.getTitle();
+        String uri = track.getUri();
+        try {
+            player.setDataSource(uri);
+            player.prepareAsync();
+        } catch (Exception e) {
+            return;
+        }
+    }
+
+    @Override
+    public void start() {
+        player.start();
+    }
+
+    @Override
+    public void pause() {
+        if (player.isPlaying()) {
+            player.pause();
+        }
+    }
+
+    @Override
+    public void stop() {
+        if (player != null) {
+            player.stop();
+        }
+    }
+
+    @Override
+    public void release() {
+        if (player != null) {
+            player.release();
+        }
+    }
+
+    @Override
+    public void seekto(int position) {
+        if (player != null) {
+            player.seekTo(position);
+        }
+    }
+
+    @Override
+    public void next() {
+        curentTrack++;
+        if (curentTrack > (mTracks.size() - 1)) {
+            curentTrack = 0;
+            play();
+        } else {
+            play();
+        }
+    }
+
+    @Override
+    public void prev() {
+        curentTrack--;
+        if (curentTrack < 0) {
+            curentTrack = mTracks.size() - 1;
+            play();
+        } else {
+            play();
+        }
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        stopForeground(true);
```

Comment:
WARNING: [CallSuperLast] super should be called at the end of the method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/android.html#CallSuperLast